test_runner/regress/test_tenants.py

```diff
@@ -390,7 +390,8 @@ def test_create_churn_during_restart(neon_env_builder: NeonEnvBuilder):
     # Tenant creation requests which arrive out of order will generate complaints about
     # generation nubmers out of order.
     env.pageserver.allowed_errors.append(".*Generation .+ is less than existing .+")
-    env.pageserver.allowed_errors.append(".*due to stale generation.+")
+    env.pageserver.allowed_errors.append(".*due to stale generation.*")
```

Comment:
Let's remove this line then:
```rust
env.pageserver.allowed_errors.append(".*due to stale generation.*")
```
Because it's misleading ("due to stale generation" message is strocon only)